style.css

```diff
@@ -1,15 +1,343 @@
   background-color: #0e0e0e;
-  font-family: 'Roboto', sans-serif;
+  font-family: "Roboto", sans-serif;
 }
 
 .mainContainer {
   display: flex;
   flex-direction: column;
   margin: 0 auto;
   justify-items: center;
+  width: 100%;
+  position: relative;
+  background-color: #ffffff;
+  font-size: 16px;
+  padding: 0 1em;
+}
+
+.head-meal-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 0.5em;
+  font-weight: bolder;
+  line-height: 1em;
+  font-size: 2.2em;
+  color: #0a0a0a;
+}
+
+.red-line-title {
+  color: #e31c3a;
+}
+
+.under-head-meal-section {
+  text-align: center;
+  font-size: 0.9em;
+}
+
+.about-you {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 4em;
+}
+.vege-card {
+  background-color: #ffffff;
+  width: 27%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.rotateimg180 {
+  transform: rotate(90deg);
+}
+.title-about-you {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  margin: 2em 0.5em;
+}
+.vege-text {
+  font-size: 0.75em;
+  margin-top: 1em;
+}
+
+.meals {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 6em;
+}
+
+.refugees {
+  background-color: #ffffff;
+  width: 27%;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: left;
+  margin-top: 3em;
+}
+
+.refugees-title {
+  color: #0a0a0a;
+  font-size: 2.2em;
+  text-align: left;
+  font-weight: bold;
+  line-height: 1em;
+  margin-left: 0.15em;
+}
+
+.text-refugees {
+  font-size: 0.75em;
+  margin: 1.5em 0.5em 4em 0.5em;
+}
+
+.meal-image {
+  color: #0a0a0a;
+  justify-content: center;
+  align-items: left;
+  margin: 0em 0.5em;
+}
+
+.button-contact {
+  margin: 1em 0;
+  border: 1px solid #e31c3a;
+  border-radius: 4px;
+  background-color: #e31c3a;
+  padding: 0.8em;
+  color: #ffffff;
+  outline: none;
+  cursor: pointer;
+  margin-left: 0.26em;
+  text-transform: uppercase;
   width: 60%;
+}
+
+.meal-block {
   background-color: #ffffff;
-  height: 100vh;
+  width: 27%;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+}
+
+.price-with-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+  margin: 1.5em 0.5em 0.5em 3.5em;
+}
+.price {
+  display: flex;
+  flex-direction: row;
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.dolar {
+  font-size: 0.7em;
+}
+
+.recipe-title {
+  font-size: 1.5em;
+  text-transform: uppercase;
+  margin-bottom: 0.5em;
+}
+.recipe-text {
+  font-size: 0.75em;
+}
+
+.third-section {
+  display: flex;
+  flex-direction: row;
+  margin-top: 5em;
+}
+
+.meal-photo {
+  display: flex;
+  width: 53%;
+  justify-content: center;
+  margin-left: -1em;
+}
+
+.city {
+  display: flex;
+  flex-direction: column;
+  width: 47%;
+  justify-content: center;
+}
```

Comment:
This whole section should probably be done differently. 

`margin-left: -1em;` seems like a hack to move an element just slightly, and using 53% and 47% is not something that's considered a best practice. 

We should discuss this (in our meeting) and find the best solution, avoiding confusion. 😺